### DISCUSSION TITLE
Github payout doesn't issue anymore

### DISCUSSION THREAD

**Discussion by ShafSpecs** [community/community#125611] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I got access to github sponsors last year and have been receiving monthly payouts for since the start of this year. I got a big one-time sponsorship on the first of May, this month (> $100), and my payout is now stuck. 

My last payout was on the 17th of April, and the next payout was slated for 22nd of May. On May 22nd, I didn't get a payout and the dashboard now says next month

```
ESTIMATED NEXT PAYOUT June 22
```

Skipping over this month entirely. Stripe dashboard mentions next month too and that my sponsorship for this month is being "processed". 

**How can I fix this please, or get information on why this month's payout was skipped totally?**

Very similar to except I previously used to get payout till this big one time sponsorship: https://github.com/orgs/community/discussions/72298

**Comment by ebndev** [comment, marked as answer]
Hi @ShafSpecs!

Just adding a comment here so the team knows that this issued has been resolved in [another discussion](https://github.com/orgs/community/discussions/125030). 😄

**Reply by cuilei5205189** [reply]
I’m experiencing a similar GitHub Sponsors payout issue.

My Sponsors dashboard estimated payout has rolled over to the next month, and in Stripe Express the balance is already available, but there is no payout pending or in transit.

I have already opened a GitHub Support ticket: #4508632, but I haven’t received a response yet.

Could you please help check this or route it to the GitHub Sponsors team? It looks like the payout was not initiated rather than delayed by the bank.

Thank you.
